playwright-e2e/page-objects/citizen/adoption-agency-and-social-worker/socialWorkerPage.page.ts

```diff
@@ -0,0 +1,82 @@
+import { Locator, Page } from '@playwright/test';
+
+import BasePage from '../../../pages/basePage.page';
+
+export class SocialWorkerPage extends BasePage {
+  readonly applicationDetailsTitle: Locator;
+
+  readonly phoneNumberText: Locator;
+  readonly phoneNumberLabel: Locator;
+
+  readonly emailAddressText: Locator;
+  readonly emailAddressLabel: Locator;
+  readonly emailAddressHint: Locator;
+
+  readonly localAuthorityInputLabel: Locator;
+  readonly localAuthorityDropdown: Locator;
+
+  readonly localAuthorityEmailText: Locator;
+  readonly localAuthorityEmailLabel: Locator;
+  readonly localAuthorityEmailHint: Locator;
+
+  readonly contactUsForHelpDropdownLink: Locator;
+  readonly contactACourtHeading: Locator;
+  readonly findACourtLink: Locator;
+
+  readonly errorNameOfChildSocialWorkerSummary: Locator;
+  readonly errorEnterUKPhoneNumberSummary: Locator;
+  readonly errorEmailFormatOptionalEmailSummary: Locator;
+  readonly errorNoEmailSummary: Locator;
+  readonly errorEnterLocalAuthoritySummary: Locator;
+  readonly errorNonGovernmentEmail: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.applicationDetailsTitle = page.getByText('Application details');
+
+    this.phoneNumberText = page.getByText('Phone number');
+    this.phoneNumberLabel = page.getByRole('textbox', { name: 'Phone number' });
+
+    this.emailAddressText = page.getByText('Email address (if known)');
+    this.emailAddressLabel = page.getByRole('textbox', { name: 'Email address (if known)' });
+    this.emailAddressHint = page.getByText('The email address should be an official government email');
+
+    this.localAuthorityInputLabel = page.locator('#location-picker');
+    this.localAuthorityDropdown = page.locator('.autocomplete__menu');
+
+    this.localAuthorityEmailText = page.getByText('Local authority email address');
+    this.localAuthorityEmailLabel = page.getByRole('textbox', { name: 'Local authority email address' });
+    this.localAuthorityEmailHint = page.getByText('This will be used to send a notification to the local authority');
+
+    this.contactUsForHelpDropdownLink = page.getByText('Contact us for help');
+    this.contactACourtHeading = page.getByRole('heading', { name: 'Contact a court that deals' });
+    this.findACourtLink = page.getByRole('link', { name: 'Find a Court or Trbunal' });
```

Comment:
Typo in Tr***i***bunal
```suggestion
    this.findACourtLink = page.getByRole('link', { name: 'Find a Court or Tribunal' });
```

Reply:
typo corrected